packages/ckb-indexer/src/indexer.ts

```diff
@@ -241,7 +244,7 @@ export class CkbIndexer implements Indexer {
       });
   }
 
-  private scheduleLoop(timeout = 1) {
+  scheduleLoop(timeout = 1): void {
```

Comment:
any reason for removing the `private` symbol modifier here?

Reply:
need to be stubbed so that I can write a test for `subscribe` method

Reply:
i see, but maybe we can add a `ts-ignore` for stub cases. we should only expose APIs that are useful to developers to avoid confusion for developers